config/acl.js

```diff
@@ -3,14 +3,12 @@ var whitelist = require('./whitelist.js');
 
 // Express middleware to check for acl privileges based on a whitelist
 var acl = {
-
   // Check if the user has admin privileges
   isAdmin: function(req, res, next) {
```

Comment:
New (definitely working) middleware to detect an admin. Requires a password in the body